design/authenticators-status/authn-status-new-implem.md

```diff
@@ -0,0 +1,109 @@
+# Implementing Authenticator Status Checks
```

Comment:
Please add a note somewhere in the beginning that devs should read the epic before implementing this and refer to the epic https://github.com/cyberark/conjur/issues/1062